src/chapters/more-flask/reusing-content.rst

```diff
@@ -289,7 +289,7 @@ insert different HTML code. It is possible to build a base template with more
 of these spaces. However, that discussion is beyond the scope of this course.
 
 If you would like to research this topic on your own, a good place to start is
-the `Jinja2 documentation <https://jinja.palletsprojects.com/en/2.11.x/templates/#template-inheritance>`__.
+the `Jinja3 documentation <https://jinja.palletsprojects.com/en/2.11.x/templates/#template-inheritance>`__.
 
 Video Summary
```

Comment:
I changed the video title in YouTube to just reference "Jinja" template extensions.  I did find one brief spot in the video that shows the link to the Jinja2 documentation.  However, it's not a big enough error to warrant redoing the whole clip or adding an admonition box.

Reply:
Cool, that works